content/cloud/custom-domain.md

```diff
@@ -0,0 +1,42 @@
+title = "Custom Domains"
+template = "cloud_main"
+date = "2023-07-26T12:00:00Z"
+enable_shortcodes = true
+[extra]
+url = "https://github.com/fermyon/developer/blob/main/content/cloud/custom-domains.md"
+
+---
+- [Relationship Between Custom Domains and Spin Applications](#relationship-between-custom-domains-and-spin-applications)
+- [Next Steps](#next-steps)
+- [Terminology](#terminology)
+
+Every Fermyon Cloud application has a default domain name in the format `<app-name>-<random-string>.fermyon.app`. The Fermyon Cloud Custom Domains feature provides a way for you to use your own domain names; providing your users with human-friendly access to your Fermyon Cloud applications. Another advantage of using Fermyon Cloud Custom Domains is that your Spin application can have a stronger sense of branding and identity. For example, for [Finicky Whiskers: The World's Most Adorable Manual Load Generator](https://www.fermyon.com/blog/finicky-whiskers-part-1-intro) we have moved away from using the default domain name `finicky-whiskers-6cavavfh.fermyon.app`, and we much prefer using the custom domain `finickywhiskers.com`. 
+
+If you do not own a domain (or are **not** interested in purchasing one) but would like to customize your Spin application's domain name while still keeping the `fermyon.app` apex domain, then Custom Fermyon Subdomains may be a good option for you. Check out the [Custom Fermyon Subdomains tutorial](./custom-fermyon-subdomain.md) for more instructions. 
+
+If you do not own a domain and you **are** interested in purchasing one, you can purchase one through a domain registrar like [Namecheap](https://www.namecheap.com/) or [Google Domains](https://domains.google). At this time, Fermyon Cloud is not a domain registrar meaning you'll have to use an external registrar.
+
+>> If you are unsure what some of the terms on this page mean, please see the section called [Terminology](#terminology) at the bottom of this page.
+
+To take advantage of Fermyon Cloud Custom Domain, you must delegate your domain to Fermyon Cloud. For a full set of instructions, please follow the [Ferymon Cloud Custom Domains tutorial](./custom-domains-tutorial.md). 
```

Comment:
I think this merits its own subhead, because it's the only how-to information currently available, and it's easy to miss (well, at least, it was easy for me!):

```suggestion
## Setting Up Custom Domains

To take advantage of Fermyon Cloud Custom Domains, you must delegate your domain to Fermyon Cloud. For a full set of instructions, please follow the [Ferymon Cloud Custom Domains tutorial](./custom-domains-tutorial.md). 
```

Reply:
Love making it super clear!